src/components/shared/forms/DateRange.vue

```diff
@@ -235,11 +253,15 @@
                   },
                   onShow: () => {
                       this.buildCal()
+                      this.prevStart = JSON.parse(JSON.stringify(this.localStart))
+                      this.prevEnd = JSON.parse(JSON.stringify(this.localEnd))
```

Comment:
Is this to avoid reactivity?
I have found lodash's cloneDeep method more reliable.